hot.js

```diff
@@ -103,21 +103,21 @@ function check(options) {
                 .catch((err) => {
                     const status = module.hot.status();
                     if (['abort', 'fail'].indexOf(status) >= 0) {
-                        //Do not modify message - CLI depends on this exact content to determine hmr operation status.
-                        log.warn(`Cannot apply update with hmr hash ${currentHash}.`);
-                        log.warn(err.stack || err.message);
+                        // Do not modify message - CLI depends on this exact content to determine hmr operation status.
+                        log.error(`Cannot apply update with hmr hash ${currentHash}.`);
+                        log.error(err.message || err.stack);
```

Comment:
Even better - print both.

Reply:
I don't think it's needed - messages look descriptive enough, i.e.:
```
CONSOLE INFO file:///app/vendor.js:72111:36: HMR: Checking for updates to the bundle with hmr hash 1a369033d922096d2138.
CONSOLE WARN file:///app/vendor.js:72112:36: HMR: Ignored an update to unaccepted module:
CONSOLE WARN file:///app/vendor.js:72112:36: HMR:          ➭ ./main.ts
CONSOLE ERROR file:///app/vendor.js:72113:38: HMR: Cannot apply update with hmr hash 1a369033d922096d2138.
CONSOLE ERROR file:///app/vendor.js:72113:38: HMR: Aborted because ./main.ts is not accepted
Update propagation: ./main.ts
```

Reply:
If you say so.